nft_ingester/src/program_transformers/bubblegum/mint_v1.rs

```diff
@@ -86,92 +89,38 @@ where
-
-                // Set initial mint info.
-                let asset_model = asset::ActiveModel {
-                    id: Set(id_bytes.to_vec()),
-                    owner_type: Set(OwnerType::Single),
-                    frozen: Set(false),
-                    tree_id: Set(Some(tree_id.clone())),
-                    specification_version: Set(Some(SpecificationVersions::V1)),
-                    specification_asset_class: Set(Some(SpecificationAssetClass::Nft)),
-                    nonce: Set(Some(nonce as i64)),
-                    royalty_target_type: Set(RoyaltyTargetType::Creators),
-                    royalty_target: Set(None),
-                    royalty_amount: Set(metadata.seller_fee_basis_points as i32), //basis points
-                    asset_data: Set(Some(id_bytes.to_vec())),
-                    slot_updated: Set(Some(slot_i)),
-                    ..Default::default()
-                };
+                unprotected_upsert_asset_data(
+                    txn,
+                    id_bytes.to_vec(),
+                    chain_mutability,
+                    chain_data_json,
+                    uri.clone(),
+                    Mutability::Mutable,
+                    JsonValue::String("processing".to_string()),
+                    slot_i,
+                    Some(true),
+                    name.to_vec(),
+                    symbol.to_vec(),
+                )
+                .await?;
 
-                // Upsert asset table base info.
-                let mut query = asset::Entity::insert(asset_model)
-                    .on_conflict(
-                        OnConflict::columns([asset::Column::Id])
-                            .update_columns([
-                                asset::Column::OwnerType,
-                                asset::Column::Frozen,
-                                asset::Column::SpecificationVersion,
-                                asset::Column::SpecificationAssetClass,
-                                asset::Column::RoyaltyTargetType,
-                                asset::Column::RoyaltyTarget,
-                                asset::Column::RoyaltyAmount,
-                                asset::Column::AssetData,
-                            ])
-                            .to_owned(),
-                    )
-                    .build(DbBackend::Postgres);
+                // Upsert into `asset` table.
 
-                // Do not overwrite changes that happened after the asset was decompressed.
-                query.sql = format!(
-                    "{} WHERE excluded.slot_updated > asset.slot_updated OR asset.slot_updated IS NULL",
-                    query.sql
-                );
-                txn.execute(query)
-                    .await
-                    .map_err(|db_err| IngesterError::AssetIndexError(db_err.to_string()))?;
+                // Set base mint info.
+                let tree_id = bundle.keys.get(3).unwrap().0.to_vec();
```

Comment:
Can this come from blockbuster instead?

Reply:
https://github.com/metaplex-foundation/blockbuster/pull/31

Reply:
Will incorporate in https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/pull/134 after the above is merged.